src/overlays/actors/ovl_Demo_Sa/z_demo_sa.c

```diff
@@ -40,16 +41,33 @@ void func_8098FC9C(DemoSa* this, GlobalContext* globalCtx);
 void func_8098FCD4(DemoSa* this, GlobalContext* globalCtx);
 void func_8098FD0C(DemoSa* this, GlobalContext* globalCtx);
 
-void func_8098FEA8(DemoSa* this, GlobalContext* globalCtx);
-void func_8098FEB4(DemoSa* this, GlobalContext* globalCtx);
-void func_8098F1C0(DemoSa* this, GlobalContext* globalCtx);
-
-static UNK_PTR D_809900E0[] = {
-    0x06002F48, 0x06003C48, 0x06003848, 0x06004848, 0x06004E48,
+void DemoSa_DrawNothing(DemoSa* this, GlobalContext* globalCtx);
+void DemoSa_DrawOpa(DemoSa* this, GlobalContext* globalCtx);
+void DemoSa_DrawXlu(DemoSa* this, GlobalContext* globalCtx);
+
+typedef enum {
+    /* 0 */ SARIA_EYE_OPEN,
+    /* 1 */ SARIA_EYE_HALF,
+    /* 2 */ SARIA_EYE_CLOSED,
+    /* 3 */ SARIA_EYE_SUPRISED,
+    /* 4 */ SARIA_EYE_SAD,
```

Comment:
Trailing comma is giving a warning, please remove it.
```suggestion
    /* 4 */ SARIA_EYE_SAD
```